scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1324,6 +1324,15 @@ Function SetEnvironmentVariable
     $env:Path="$env:Path;C:\Program Files\iotedge-moby;C:\Program Files\iotedge"
 }
 
+Function SetupMountedStorage
```

Comment:
Here you don't seem to delete the mount folder if it exists.

Reply:
The Setup-Env.ps1 should handle that part

Reply:
But we should probably keep all the logic in one place, so I will move